NppMenuSearch/Localization/NppMenuSearch.dll.italian.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<NppMenuSearch>
+	<Native-Lang name="Italian" version="2025-06-08">
+		<General>
+			<SearchWidgetTitle>Ricerca Notepad++</SearchWidgetTitle>
+		</General>
+		<MenuTitles>
+			<RepeatPreviousCommand>Ripeti il comando precedente</RepeatPreviousCommand>
+			<RepeatCommand>Ripeti comando “{0}”</RepeatCommand>
+			<About>&amp;Informazioni su...</About>
+			<FixWidgetSize>&amp;Correggi la dimensione del widget</FixWidgetSize>
```

Comment:
"Corriggi" sounds like "to correct" (?). That was not indented. I changed the English translation to "Lock widget size" and added comments to describe the purpose into NppMenuSearch.dll.english.xml.
![Widget size locked](https://github.com/user-attachments/assets/f694412a-b8ed-47b7-abf7-72737cd710c3) ![Widget size not locked](https://github.com/user-attachments/assets/068c6818-983c-49d9-b5de-a9ff920313ae)